data/101/753/931/101753931.geojson

```diff
@@ -36,7 +36,7 @@
         "Voluntari"
     ],
     "name:eng_x_preferred":[
-        "Parcu Voluntarilor"
+        "Voluntari"
```

Comment:
In this case the new English preferred name now matches the wof:name, so no updated needed there.